openedx/core/djangoapps/agreements/serializers.py

```diff
@@ -17,4 +17,16 @@ class IntegritySignatureSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = IntegritySignature()
-        fields = ('username', 'course_id', 'created_at')
```

Comment:
NIT: This line needs to be returned, otherwise tests will fail for the Integrity Signature view with these messages:

```
=========================== short test summary info ============================
FAILED openedx/core/djangoapps/agreements/tests/test_views.py::IntegritySignatureViewTests::test_200_get_for_admin - AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the IntegritySignatureSerializer serializer.",)
FAILED openedx/core/djangoapps/agreements/tests/test_views.py::IntegritySignatureViewTests::test_200_get_for_course_staff_request - AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the IntegritySignatureSerializer serializer.",)
FAILED openedx/core/djangoapps/agreements/tests/test_views.py::IntegritySignatureViewTests::test_200_get_for_user_request - AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the IntegritySignatureSerializer serializer.",)
FAILED openedx/core/djangoapps/agreements/tests/test_views.py::IntegritySignatureViewTests::test_post_duplicate_integrity_signature - AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the IntegritySignatureSerializer serializer.",)
FAILED openedx/core/djangoapps/agreements/tests/test_views.py::IntegritySignatureViewTests::test_post_integrity_signature - AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the IntegritySignatureSerializer serializer.",)
==== 5 failed, 1792 passed, 90 skipped, 3823 warnings in 645.73s (0:10:45) =====
```